lib/bar/TopBar.js

```diff
@@ -19,29 +19,31 @@ export default class TopBar extends React.Component {
     title: PropTypes.string,
     height: PropTypes.number,
     backTitle: PropTypes.string,
-    backImage: PropTypes.any,
+    backIcon: PropTypes.any,
     onBack: PropTypes.func,
   };
 
   static defaultProps = {
     displayed: false,
     title: '',
-    backTitle: 'Back',
-    backImage: require('../../Assets/angle-left.png'),
+    backTitle: '',
+    backIcon: null,
   };
 
   renderBackButton() {
-    const { onBack, backImage } = this.props;
+    const { onBack, backIcon } = this.props;
 
     // do not display back button if there isn't a press handler
     if (onBack) {
       return (
         <TouchableOpacity style={styles.backContainer} onPress={onBack}>
-          <Image source={backImage} />
-          {Platform.OS === 'ios' &&
+          {backIcon && React.isValidElement(backIcon) && backIcon}
+          {backIcon && !React.isValidElement(backIcon) && <Image source={backIcon} />}
```

Comment:
We might introduce a new prop called `backElement` to keep this backwards compatible as well:

```suggestion
const useBackElement = !!backElement && React.isValidElement(backElement)

          {useBackElement && backElement}
          {!useBackElement && !!backImage && <Image source={backImage} />}
```